### PR TITLE
Prebid 7: update multiple adapters to standardize bcat

### PR DESCRIPTION
**Pull request by ChrisHuie**

### CODE REVIEW

**Review by mmoschovas — APPROVED** [review]

**Comment by mmoschovas** [comment]
@ChrisHuie unsure if this needs a second review because it dealt with multiple bidder adapters.

**Review by patmmccann — APPROVED** [review]

**Comment by patmmccann** [comment]
fixes #7925

**Pull request merged**
